### ISSUE TITLE
[Feature Request] Bulk Copy Firewall Rules To Interface

### ISSUE THREAD

**Issue by ky41083** [opnsense/core#7817]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [X] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [X] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Is your feature request related to a problem? Please describe.**

I would like to be able to bulk copy rules between interfaces. I can copy 1 rule at a time to another interface, but this gets tedious when there is a need to copy many rules at a time to a different interface.

**Describe the solution you like**

I would like a button (Copy selected) added to the Firewall Rules Interface page button bar that can be used after selecting the rules to be copied. This would bring up a form select (or maybe even a multi-select to allow copying to multiple interfaces at once) asking which interface the selected rules would be copied to.

**Describe alternatives you considered**

Copying 1 rule at a time is an alternative, but is a hard sell in production environments with 4+ WAN interfaces all using the same 20+ rules. Or 20+ VLAN interfaces with 5+ rules in common.

**Additional context**

<img width="776" alt="image" src="https://github.com/user-attachments/assets/bd44967c-4c0a-42ee-8a51-1fada2e2161f">

**Comment by celloza** [comment]
Would like to see this too.

**Comment by sysxtreme** [comment]
Totally necessary!

**Comment by aibix0001** [comment]
Yes, please! I'm in the process of converting from pfSense to OPNsense and this is sorely missing.

**Comment by chotaire** [comment]
This really needs to be implemented, unless companies are willing to hire employees who do nothing other than cloning firewall rules to another interface, one by one.

**Comment by Monviech** [comment]
Maybe these employees could work smarter and not harder:

https://docs.opnsense.org/manual/how-tos/security-zones.html
